components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -155,3 +154,55 @@ assertTrue(Updater.loadError["not_found"].testHasValue())
 // Does the counter have the expected value?
 assertEquals(2, Updater.loadError["not_found"].testGetValue())
 ```
+
+## Timing Distributions
+
+Used to accumulate and store time measurement, for analyzing distributions of the timing data.
+
+If you wanted to create a timing distribution to measure page load times, first you need to add an 
+entry for it to the `metrics.yaml` file:
+
+```YAML
+pages:
+  page_load:
+    type: timing_distribution
+    time_unit: millisecond
+    description: >
+      Counts how long each page takes to load
+    ...
+```
+
+Note that `time_unit` is required and allows for values like `millisecond` and `second`. Which range 
+of values is recorded in detail depends on the time_unit, e.g. for milliseconds all values greater 
+60000 are recorded as overflow values.
+
+Now that the timing distribution is defined in `metrics.yaml` you can use the metric to record data
+in the application code:
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Pages
+
+// ...
+pages.pageLoad.accumulate(1L) // Accumulates a sample of 1 millisecond
+pages.pageLoad.accumulate(10L) // Accumulates a sample of 10 milliseconds
+```
+
+There are test APIs available too.  For convenience, properties `sum` and `count` are exposed to 
+facilitate validating that data was recorded correctly.  Continuing the `pageLoad` example above,
+at this point the metric should have a `sum == 11` and a `count() == 2`:
```

Comment:
Why does `sum` have property syntax & `count()` function syntax.
Please change the API to use the same approach for both.

Reply:
Good point, I just needed to move this to a `get()` in the property so that it was a calculated property instead.